src/test/groovy/org/prebid/server/functional/tests/BidValidationSpec.groovy

```diff
@@ -212,4 +212,34 @@ class BidValidationSpec extends BaseSpec {
         def metrics = defaultPbsService.sendCollectedMetricsRequest()
         assert metrics["adapter.generic.requests.bid_validation"] == initialMetricValue + 1
     }
+
+    def "PBS should return error when between refers to unknown bidder alias"() {
+        given: "Default basic BidRequest"
+        def bidRequest = BidRequest.defaultBidRequest
+        bidRequest.ext.prebid.aliasgvlids = ["between": 1]
+        bidRequest.ext.prebid.aliases = ["appnexus": GENERIC]
```

Comment:
1) Please try to not use real names in tests.
2) How `appnexus` ties to `between`?

Reply:
Same, that comment wasn't answered here nor resolved in the code.

Reply:
1. Why don't use real names in tests?
We already used real names :)
2. Nothing ties them.